package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.0.0",
+  "version": "2.1.0",
```

Comment:
Shouldn't need to do this in a PR, when they publish a release they'll handle this.